contracts/external/cw-token-swap/src/contract.rs

```diff
@@ -88,6 +89,7 @@ fn get_counterparty<'a>(
 /// escrow funds if both counterparties have funded the contract.
 ///
 /// NOTE: The caller must verify that the denom of PAID is correct.
+#[allow(clippy::too_many_arguments)]
```

Comment:
do we need this? none of the arguments in the PR have changed. think this might be leftover from the first pass.